lib/modules/platform/codecommit/index.spec.ts

```diff
@@ -49,7 +49,19 @@ describe('modules/platform/codecommit/index', () => {
 
   it('validates massageMarkdown functionality', () => {
     const newStr = codeCommit.massageMarkdown(
-      '<details><summary>foo</summary>bar</details>text<details>\n<!--renovate-debug:hiddenmessage123-->'
+      '<!--renovate:start--><details><summary>foo</summary>bar</details>text<details>\n<!--renovate-debug:hiddenmessage123--><!--renovate:end-->'
+    );
+    expect(newStr).toBe(
+      '**foo**bartext\n[//]: # (<!--renovate-debug:hiddenmessage123-->)'
+    );
+  });
+
+  it('validates massageMarkdown functionality with new lines', () => {
```

Comment:
description is wrong, there are no newlines inside body. so what's that actually testing?

Reply:
The string contains new lines.

```
<!--renovate:start--><details><summary>foo</summary>bar</details>text<details>\n<!--renovate-debug:hiddenmessage123--><!--renovate:end-->
```

vs. 

```
`<!--renovate:start-->
<details><summary>foo</summary>bar</details>text<details>
<!--renovate-debug:hiddenmessage123-->
<!--renovate:end-->`
```

You are probably looking for explicit `\n`?